### PR TITLE
Manage external storage

### PR DESCRIPTION
**Pull request by m-kuhn**

### CODE REVIEW

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 64e9a56a0e11233da72618ee3578c1e50adeb223:
  - [**arm64_v8a**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-117b34979d5c77fc7d5b30e5714178a3ed4bdbcf-arm64_v8a.apk)

Other architectures: [armv7](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-117b34979d5c77fc7d5b30e5714178a3ed4bdbcf-armv7.apk), [x86_64](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-117b34979d5c77fc7d5b30e5714178a3ed4bdbcf-x86_64.apk), [x86](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-117b34979d5c77fc7d5b30e5714178a3ed4bdbcf-x86.apk)

File: .github/workflows/android.yml [hunk 1]
```diff
@@ -149,6 +149,7 @@ jobs:
                 -B "${CMAKE_BUILD_DIR}" \
                 -G Ninja \
                 -DCMAKE_BUILD_TYPE=${{ steps.vars.outputs.BUILD_TYPE }} \
+                -D WITH_ALL_FILES_ACCESS=ON \
```
Comment (nirvn) [inline .github/workflows/android.yml:152]:
```suggestion
                -DWITH_ALL_FILES_ACCESS=ON \
```

Comment (m-kuhn) [reply .github/workflows/android.yml:110]: Should be fine after the rebase now

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 123860818aaa3ed372a054f489b63e5aca1cfa30:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-6552daa347c9f3913e0ec72f434a415ad79b253d-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-6552daa347c9f3913e0ec72f434a415ad79b253d-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-6552daa347c9f3913e0ec72f434a415ad79b253d-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-6552daa347c9f3913e0ec72f434a415ad79b253d-x86-android.apk)

**Review by nirvn — APPROVED** [review]

**Comment by qfield-fairy** [comment]
🎉 Ta-daaa, freshly created APKs are available for 612ffa50b05b7086eafcb37ab21819c5b79de967:
  - [**arm64-android**](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-7118d0b8465ac39f91ec38123fc9ece2d6e9ad37-arm64-android.apk)

Other architectures: [arm-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-7118d0b8465ac39f91ec38123fc9ece2d6e9ad37-arm-android.apk), [x64-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-7118d0b8465ac39f91ec38123fc9ece2d6e9ad37-x64-android.apk), [x86-android](https://sos-ch-dk-2.exo.io/qfieldapks/ci-builds/qfield-dev-2478-7118d0b8465ac39f91ec38123fc9ece2d6e9ad37-x86-android.apk)

**Pull request merged**